src/main.cpp

```diff
@@ -839,10 +837,6 @@ static bool AcceptToMemoryPoolWorker(CTxMemPool &pool,
 
         // Set extraFlags as a set of flags that needs to be activated.
         uint32_t extraFlags = 0;
```

Comment:
do we still need extraflags?

Reply:
As the code stands we don't. It is a left over from the op_code activation from May 2018 upgrade. Will remove. But I guess @awemany could use it again from op_checkdatasigverify.